python/python/glide/async_commands/core.py

```diff
@@ -448,6 +448,29 @@ async def get(self, key: str) -> Optional[str]:
         """
         return cast(Optional[str], await self._execute_command(RequestType.Get, [key]))
 
+    async def getdel(self, key: str) -> Optional[str]:
+        """
```

Comment:
copy paste from the docs, no need to invent or modify the official doc:
```
Get the value of key and delete the key. This command is similar to GET, except for the fact that it also deletes the key on success (if and only if the key’s value type is a string).
```

Reply:
Consistent to Java doc.